packages/components/.storybook/preview.tsx

```diff
@@ -10,7 +10,7 @@ import { addParameters } from "@storybook/react";
 import { withA11y } from "@storybook/addon-a11y";
 
 addDecorator(withA11y);
-addDecorator((storyFn) => <div dir="ltr">{storyFn()}</div>);
+addDecorator((storyFn) => <div>{storyFn()}</div>);
```

Comment:
removing this fixes the style overriding on buttons, idk if we need it for something else though

Reply:
Removing it seems perfectly reasonable. My guess is storybook will apply this to a wrapping div on its own based on UI options.

Reply:
I think to fix the overrides we can remove the all.css imports, which are no longer necessary

Reply:
~We will want to import `<GlobalStyles />` from twin.macro though~, oh, this is pre-twin.macro right?

In that case, you can remove the rtl step at https://github.com/chanzuckerberg/lp-design-system/blob/master/packages/styles/postcss.config.js#L16

Reply:
oh nice, thanks!